### ISSUE TITLE
Creating an easyslider portlet

### ISSUE THREAD

**Issue by davidjb** [collective/collective.easyslider#19]
I'm looking to be able to add/create an easyslider portlet that works in the same manner as assigning a slider to a given folder or collection.  So, be able to configure slides and all available options except in a portlet context rather than on a folder (eg not like the current hover portlet within this package).

My thoughts are either to have:
- A portlet that allows you to select a context and display its slider inline, or
- A portlet that gives you the full slider configuration, and saves settings against that portlet.

My thinking is that I'd opt for the first option since it would help reduce duplication (eg one slider, several locations) and increase flexibility (eg can copy a folder, can't copy a portlet).

Does something for this already exist?  If not, does it make sense for me to aim to include it within collective.easyslider? 

Thoughts?

**Comment by vangheem** [comment]
Yes, I've been wanting to do this for some time. It'd be great if you could contribute this!

I personally like your second option since that wouldn't require someone to configure the slider first somewhere else. 

If you do any work on it, please just replace the existing slider portlet which is mostly useless...

**Comment by djay** [comment]
I know it's nothing to do with easyslider but we normally do this using collective.listingviews and diazo + a jquery plugin. That lets the editor create a portlet that points to a collection or the portlet context, select "slider view" and have it turned into a slider automatically. 
Then you get full control over the js code.
